### DISCUSSION TITLE
question about license terms and free features

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/xts#6] [Ideas]
@pgundlach,

I see that license in xts (BSD) differs from the one chosen in Publisher (AGPLv3).

I think that there is important from the start to specify which features are and will be available for everyone to use and those features that are not intended for general usage.

I mean, remote image loading (only to name a feature) became a paid feature.

If this is going to be the same in xts (certain features not free available), it is essential that it is clear from the start which features are not available for the general public.

Many thanks for your help.

**Comment by pgundlach** [comment]
I don't have any plans at the moment to have non-free features. 

That said, I don't give any promises at all and everything might change at any time. Still thinking about how to monetize boxes and glue / xts.

I kind of like the approach of https://github.com/squidfunk/mkdocs-material which has a sponsor-to-get-the-newest-features version. But this needs a lot of users which is quite unlikely for xts.

If someone comes up with a good idea, I'd be glad to hear it!
